scripts/bundlesize/bundlesize.config.json

```
@@ -2,7 +2,7 @@
     "files": [
         {
             "path": "packages/@lwc/engine-dom/dist/index.js",
-            "maxSize": "22KB"
+            "maxSize": "22.10KB"
```

```suggestion
            "maxSize": "22KB"
```
When I run `bundlesize`, I see
```txt
PASS: packages/@lwc/engine-dom/dist/index.js: 21.94KB <= 22.10KB
```
so this change doesn't seem necessary.